app/controllers/api/base_controller.rb

```diff
@@ -225,6 +226,24 @@ def paginate_options
       }
     end
 
+    def assign_lone_taxonomies
+      return unless resource_class_for(resource_name)
+      [Location, Organization].each do |taxonomy|
+        tax_name = taxonomy.to_s.downcase
+        if resource_class.reflections.has_key? tax_name.pluralize
+          tax_ids = "#{tax_name}_ids"
+          next if params[resource_name].try(:has_key?, tax_ids)
```

Comment:
This skips parameter filtering, unsure if this is a good thing or not. Any thoughts?

Reply:
you mean strong params? I think it's fine, it doesn't assign anything, just checks if a certain parameter has been passed by the user.